modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,172 @@
+----
+
+<1> The *fluent-plugin-remote-syslog* plug-in.
+<2> The FQDN or IP address of the syslog server.
+<3> The port number to connect on. Defaults to 514.
+<4> The name of the syslog server.
+<5> Removes the prefix from the tag. Defaults to '' (empty).
+<6> The field to set the syslog key.
+<7> The Rsyslog log facility or source.
+<8> The Rsyslog log severity.
+<9> Use the severity and facility from the record if available.
+<10> Use this field as the key to look on the record, to set the payload on the syslog message. Defaults to `message`.
+
+
+// Above definitions from https://github.com/docebo/fluent-plugin-remote-syslog
+
+
+.Sample `secure-forward` ConfigMap based on the configuration
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: secure-forward
+  namespace: openshift-logging
+  selfLink: /api/v1/namespaces/openshift-logging/configmaps/secure-forward
+  uid: 6fb09e64-0821-4939-b4bb-71b8100477dc
+  resourceVersion: '30996'
+  creationTimestamp: '2020-02-10T18:38:32Z'
+data:
+  secure-forward.conf: |
+    <store>
+     @type syslog_buffered
+     remote_syslog rsyslogserver.openshift-logging.svc.cluster.local
+     port 514
+     hostname fluentd-4nzfz
+     remove_tag_prefix=tag
+     tag_key ident,systemd.u.SYSLOG_IDENTIFIER
+     facility local0
+     severity info
+     use_record true
+     payload_key=message
+    </store> 
+----
+
+.Procedure
+
+To configure the *fluent-plugin-remote-syslog* plug-in:
+
+. Create a configuration file named `secure-forward.conf` containing the following
+parameters within the `<store>` stanza: 
+
+.. Specify `@type: syslog_buffered` to invoke the *fluent-plugin-remote-syslog* plug-in:
++
+----
+@type syslog_buffered <1>
+----
++
+<1> Specify the plug-in to use, either: `syslog` or `syslog_buffered`. 
+
+.. Configure the name, host, and port for your external Fluentd server:
++
+[source,yaml]
+----
+remote_syslog <fqdn or IP> <1>
+port <number> <2>
+hostname <name> <3>
+----
++
+<1> Specify the FQDN or IP address of the Rsyslog instance.
+<2> Specify the port of the Rsyslog instance.
+<3> Specify the name of the Rsyslog instance.
+
+If you specify two or more receivers, `out_secure_forward` uses these server nodes in a round-robin order.
++
+For example:
++
+[source,yaml]
+----
+ remote_syslog rsyslogserver.openshift-logging.svc.cluster.local <2>
+ port 514 <2>
+ hostname fluentd-server <4>
+----
+
+.. Configure the other Rsyslog varaibles as needed:
++
+----
+remove_tag_prefix <1>
+tag_key <key> <2>
+facility <value>  <3>
+severity <value>  <4>
+use_record <value> <5>
+payload_key=message <6>
+----
++
+<1> Enter this value to remove the `tag` field from the syslog prefix.
+<2> Specify the field to set the syslog key.
+<3> Specify the Rsyslog log facility or source. For values, see link:https://tools.ietf.org/html/rfc5424#section-6.2.1[RTF 5424].
+<4> Specify the Rsyslog log severity. For values, see link:https://tools.ietf.org/html/rfc5424#section-6.2.1[RTF 5424].
```

Comment:
Should link to RFC3164 instead.  I don't believe the current fluent-plugin-remote-syslog supports RFC5424